src/main/java/seedu/address/logic/parser/CliSyntax.java

```diff
@@ -12,5 +12,6 @@ public class CliSyntax {
     public static final Prefix PREFIX_ADDRESS = new Prefix("a/");
     public static final Prefix PREFIX_PARENT_PHONE = new Prefix("c/");
     public static final Prefix PREFIX_TAG = new Prefix("t/");
+    public static final Prefix PREFIX_TASK_TITLE = new Prefix("t/");
```

Comment:
Overall looks great! This might not be an issue, but will it be better if we use another prefix name to avoid any confusion to the users?

Reply:
@CarrieLi1015 I think we discussed about this previously that it doesn't really matter as the tag prefix and task title prefix will never be used together and the formats are clearly stated in our user guide as well. But we can talk about this during our meeting too, and see what the others think. 😄